.travis.yml

```
@@ -10,6 +10,12 @@ jobs:
       install: "pip install -r requirements.txt"
       script: sh run_tests.sh
 
+    - language: python
+      python: 3.9
+      cache: pip
+      install: "pip install $(pipenv lock --requirements)"
```

Once https://github.com/fedora-infra/mote/pull/145#pullrequestreview-517871708 is followed, proceed to the following.
```suggestion
      install: "pip install -r docs/requirements.txt"
```

I noticed that the `requirements.txt` is not updated to address #145. It is understandable that the `Pipfile.lock` now addresses the requirements but we would want to
- First, maintain the requirements for the docs and the projects separately. (Cleaner that way)
- Second, remove the `requirements.txt` (from project root) altogether to avoid confusions. (The `pipenv lock --requirements` command would generate one during the build anyway)

---

Hmm… are you sure that is what we want to do? 😀 

Pipenv provides a comprehensive way to manage virtualenvs. Setting up other virtualenvs on top of the existing Pipenv environment goes against the principle of how Pipenv works. By adding the docs dependencies as **development dependencies**, this allows us to install (or not) the development dependencies together in the same virtualenv.

This is particularly useful when going back and forth between writing code and documentation.

---

Umm... I guess that makes sense. A singular lock file for both documentation and code would be convenient in this case.

But then again, if we are using the lock file - we are generating a `requirements.txt` during the build. We would want to remove the `requirements.txt` file.

That is where the confusion rose from, I think. I checked the `requirements.txt` file and it did not seem to have the dependencies for the documentation while the lock file did.